### ISSUE TITLE
Activation email placeholders are not parsed if use file chunk

### ISSUE THREAD

**Issue by Uch99** [modxcms/Login#162]
Activation email placeholders are not parsed if use activationEmailTplType = file  
Login 1.9.7 + MODX Revo 2.7.2

**Comment by sdrenth** [comment]
@Uch99 Can confirm this issue.

In the getChunk method of login.class.php change the file case to this would resolve this issue:
```
case 'file':
    $name = str_replace(array(
        '{base_path}',
        '{assets_path}',
        '{core_path}',
    ),array(
        $this->modx->getOption('base_path'),
        $this->modx->getOption('assets_path'),
        $this->modx->getOption('core_path'),
    ),$name);
    $this->modx->setPlaceholders($properties);

    $chunk = $this->modx->newObject('modChunk');
    $chunk->setContent(file_get_contents($name));
    $chunk->setCacheable(false);
    $output = $chunk->process($properties);
    break;  
```

**Comment by Jako** [comment]
Fixed with 1.9.8
